django-cloudlaunch/baselaunch/domain_model.py

```diff
@@ -55,5 +55,16 @@ def get_cloud_provider(cloud, cred_dict):
         config.update(cred_dict)
         return CloudProviderFactory().create_provider(ProviderList.AZURE,
                                                       config)
+    elif isinstance(cloud, models.GCE):
+        config = {
+            'gce_client_email': cred_dict['client_email'],
```

Comment:
I agree, this was also my question in issue 45. If you're fine with it I can go ahead and make the change.